io/io/src/TBufferFile.cxx

```diff
@@ -196,7 +196,7 @@ void TBufferFile::ReadTString(TString &s)
       else
          nbig = nwh;
 
-      s.Clobber(nbig);
+      nbig = s.Clobber(nbig);
```

Comment:
Maybe a comment here to clarify the intent of changing the value of the `nbig` variable for the rest of the function (since it wasn't done previously)?